api/src/main/java/com/nicknathanjustin/streamercontracts/users/UserModel.java

```diff
@@ -0,0 +1,30 @@
+package com.nicknathanjustin.streamercontracts.users;
+
+import lombok.*;
```

Comment:
super minor, but i think turning off start importats is better than leaving them on. This is a configuration in intellij. If you're okay with this, follow this link to disable star imports -> https://stackoverflow.com/questions/3587071/disable-intellij-starred-package-imports